plugins/serializers/graphite/graphite.go

```diff
@@ -316,3 +326,12 @@ func sanitize(value string) string {
 	// Replace any remaining illegal chars
 	return allowedChars.ReplaceAllLiteralString(value, "_")
 }
+
+func sanitizeTag(name string, value string) string {
```

Comment:
Again wondering about clarity of naming thats more descriptive of the difference strictSanitize for the one above and sanitize for the new one perhaps?

Reply:
Named each Sanitize function to match what it does (`strict` versus `compatible`).